src/EnergyPlus/PlantLoadProfile.cc

```diff
@@ -258,6 +326,7 @@ void PlantProfileData::UpdatePlantProfile(EnergyPlusData &state) const
 
     // Set outlet node variables that are possibly changed
     state.dataLoopNodes->Node(this->OutletNode).Temp = this->OutletTemp;
+    state.dataLoopNodes->Node(this->OutletNode).MassFlowRate = this->MassFlowRate;
 }
```

Comment:
Component models should not be setting plant node mass flow rates directly as is done here.  You already use central routine SetComponentFlowRate and it sets the node flow rates.  There could be other limitations outside of the component model that are affecting flow.  Managing that larger system gets unwieldly if component models do this sort of thing.